types/command-line-args/index.d.ts

```diff
@@ -25,50 +25,95 @@
  */
 declare function commandLineArgs(optionDefinitions: commandLineArgs.OptionDefinition[], options?: commandLineArgs.Options): any;
 
-declare module commandLineArgs { 
+declare module commandLineArgs {
 
-    export interface OptionDefinition {
-        /**
-         * The only required definition property is name, the value of each option will be either a Boolean or string.
-         */
-        name: string,
-        /**
-         * The type value is a setter function (you receive the output from this), enabling you to be specific about the type and value received.
-         */
-        type?: (arg: string) => any,
-        /**
-         * getopt-style short option names. Can be any single character (unicode included) except a digit or hypen.
-         */
-        alias?: string,
-        /**
-         * Set this flag if the option takes a list of values. You will receive an array of values, each passed through the type function (if specified).
-         */
-        multiple?: boolean,
-        /**
-         * Any unclaimed command-line args will be set on this option. This flag is typically set on the most commonly-used option to make for more concise usage (i.e. $ myapp *.js instead of $ myapp --files *.js).
-         */
-        defaultOption?: boolean,
-        /**
-         * An initial value for the option.
-         */
-        defaultValue?: any,
-        /**
-         * When your app has a large amount of options it makes sense to organise them in groups.
-         * There are two automatic groups: _all (contains all options) and _none (contains options without a group specified in their definition).
-         */
-        group?: string | string[],
-    }
+  export interface OptionDefinition {
+    /**
+     * The only required definition property is name, the value of each option will be either a Boolean or string.
+     *
+     * @type {string}
```

Comment:
These type annotations are redundant, so you should leave them out. I don't think `@memberof` is a good idea either, since it's a property that's statically evident from the scope of these properties. I don't think Document This is helpful here.